scripts/install_requirements.py

```diff
@@ -42,7 +50,10 @@ def _build_argument_parser():
 
 
 def _process_line(src_line):
-    return src_line.replace(">=", "==")
+    if python_version in _PINNED_REQUIREMENTS and src_line.startswith("numpy"):
+        return _PINNED_REQUIREMENTS[python_version]
+    else:
+        return src_line.replace(">=", "==")
```

Comment:
@riedgar-ms and @adrinjalali how about this as a transitional solution before we move towards something more sustainable? I don't have the time to invest at the moment, and I would really like to get rid of the red in CI as it makes the daily monitoring tedious. I started by implementing a more elaborate solution creating a new flag to get the python version input from the CI conf files, enabling us also more convenient testing, but I it felt like an overkill for something we want to change. I also created more flexible dictionaries and code that can be modified easily to support more packages than `numpy`, but at the moment we don't need that either and we might not in the near future. It is really just for the `--pinned` case which isn't something a user will use.